components/docs-chef-io/content/automate/chef_infra_in_chef_automate.md

```diff
@@ -0,0 +1,67 @@
+| opscode_erchef['depsolver_pooler_timeout'] 	    | 100000	                                  | 0	                               | ```erchef.v1.sys.depsolver.pool_queue_timeout```           |
+| opscode_erchef['depsolver_pool_queue_max'] 	    | 10	                                      | 50	                             | ```erchef.v1.sys.depsolver.pool_queue_max```               |
+| opscode_erchef['db_pool_size'] 	                | 40	                                      | 20	                             | ```erchef.v1.sys.sql.pool_max_size```                      |
+| opscode_erchef['db_pool_queue_max'] 	          | 40	                                      | 20	                             | ```erchef.v1.sys.sql.pool_queue_max```                     |
+| opscode_erchef['ibrowse_max_pipeline_size']	    | 1	                                        | 1		                             | ```erchef.v1.sys.ibrowse.max_pipeline_size```              |
+| opscode_erchef['ibrowse_max_sessions']	        | 256	                                      | 256		                           | ```erchef.v1.sys.ibrowse.max_sessions```                   |
+| opscode_erchef['max_request_size']	            | 2000000	                                  | 2000000	                         | ```erchef.v1.sys.api.max_request_size```                   |
+| opscode_erchef['keygen_cache_size'] 	          | 1000	                                    | 10	                             | ```erchef.v1.sys.keygen.cache_size```                      |
+| opscode_erchef['reindex_batch_size']	          | 10	                                      | 10	                             | ```erchef.v1.sys.index.reindex_batch_size```               |
+| opscode_erchef['reindex_sleep_min_ms']	        | 500	                                      | 500	                             | ```erchef.v1.sys.index.reindex_sleep_min_ms```             |
+| opscode_erchef['reindex_sleep_max_ms']	        | 2000	                                    | 2000	                           | ```erchef.v1.sys.index.reindex_sleep_max_ms```             |
+| opscode_erchef['reindex_item_retries']	        | 3	                                        | 3	                               | ```erchef.v1.sys.index.reindex_item_retries```             |
+| opscode_erchef['cbv_cache_enabled']	            | FALSE	                                    | FALSE	                           | ```erchef.v1.sys.api.cbv_cache_enabled```                  |
+| opscode_erchef['search_queue_mode']	            | batch	                                    | batch	                           | ```erchef.v1.sys.index.search_queue_mode```                |
+| oc_chef_authz['http_queue_max'] 	              | 200		                                    | -                                | ```erchef.v1.sys.authz.pool_queue_max```                   |	
```

Comment:
@kalroy what is the default value for automate HA, I have kept '-' for now?

Reply:
Use the same value as the Standalone one